apps/sms/test/unit/contacts_test.js

```diff
@@ -527,10 +527,10 @@ suite('Contacts', function(done) {
     });
 
     test('The mozContacts find() call returned an error', function(done) {
-      Contacts.findByPhoneNumber('callonerror').then(
-        () => done(new Error('The promise should be rejected.')),
-        () => done()
-      );
+      Contacts.findByPhoneNumber('callonerror').then((results) => {
```

Comment:
Quite severe regression actually and we both didn't manage to catch it during review :) How do you feel about simple integration test for this bug?